GrassChoppers/Modules/Home/HomeLogicController.swift

```diff
@@ -25,11 +26,21 @@ class HomeLogicController: HomeLogicControlling {
         self.dataManager.delegate = self
     }
     
+    func viewDidLoad() {
+        
+        dataManager.requestInitialState()
+        
+    }
+    
     func buttonPressed() {
         router.routeToNextScreen()
```

Comment:
Less important, but this is the logic controller telling the router that it is time to move to the next screen.